templates/models/deployment.yaml

```diff
@@ -45,10 +45,8 @@ spec:
 {{- if .env }}
             {{- toYaml .env | nindent 12 }}
 {{- end }}
-{{- if .resources }}
           resources:
-            {{- toYaml .resources | nindent 12 }}
-{{- end }}
+            {{- toYaml .Values.models.resources | nindent 12 }}
```

Comment:
i believe this will not work, since we are inside the `{{- range $key, $val := .Values.models }}` section the `.` refers to the element in the range. I think you either need $.Values... or .resources. I think the latter is better since this allows me to set the resources on a per model basis.

```
models:
  biocro-0-95:
    enabled: true
    image: model-biocro-0.95
    replicaCount: 1
    resources:
      limits:
        cpu: "1"
```

Reply:
Agreed, closing this PR. We will keep the second approach to set resources in models.